src/index.js

```diff
@@ -10,8 +10,9 @@ import './unveil';
 // this story, so that other interactives can opt to wait for Odyssey to load
 // before trying to touch the DOM (mounts, decoys, etc.)
 window.__IS_ODYSSEY_FORMAT__ = true;
+window.__ODYSSEY_EXEC__ = null;
 
-proxy('odyssey').then(() => {
+const go = async () => {
```

Comment:
This `async` isn't needed here because we're returning a promise below, which does the same thing.

Also interactives should still use `whenOdysseyLoaded` so it doesn't really do anything.